### PR TITLE
[IMP] Add start and end date in task form view

### PR DESCRIPTION
**Pull request by Doscaal**
Start and end date are used by timeline view but they don't appear in form view. It's seems to be interesting to see this info on form view on top of timeline view

### CODE REVIEW

**Comment by Doscaal** [comment]
The CLA issue are normaly fixed

**Comment by pedrobaeza** [comment]
Please change module version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

If you have sent the CLA, we'll check it soon.

File: project_timeline/__manifest__.py [hunk 1]
```diff
@@ -6,10 +6,11 @@
 {
     "name": "Project timeline",
     "summary": "Timeline view for projects",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.1.0",
     "category": "Project Management",
     "website": "https://www.tecnativa.com",
     "author": "Tecnativa, "
+              "Doscaal, "
```
Comment (pedrobaeza) [inline project_timeline/__manifest__.py]: Please add yourself on README on contributors section, but this minimum change is not too significant for making you be part of coauthors.

**Review by dreispt — APPROVED** [review]
We have an ICLA.
You can add your name to the Contributors section of the README file.

**Comment by Doscaal** [comment]
I remove Doscaal from manifest and I add it on README

**Review by pedrobaeza — APPROVED** [review]
Thank you very much

**Review by rafaelbn — APPROVED** [review]
Tested in runbot 👍

**Pull request merged**